src/modules/submodules/consensus.py

```diff
@@ -330,7 +328,7 @@ def _get_slot_delay_before_data_submit(self, blockstamp: BlockStamp) -> int:
 
         members, _ = consensus_contract.functions.getMembers().call(block_identifier=blockstamp.block_hash)
 
-        mem_position = members.index(variables.ACCOUNT.address)
+        mem_position = members.index(variables.ACCOUNT.address)  # doesn't supposed to be used without private key provided?
```

Comment:
Yeah, seems this is bug.
Definitely we should check if ACCOUNT is not None here.

Reply:
Done